spec/models/user_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  let(:user) {FactoryBot.create :user}
+
+  subject {user}
+
+  it "has a valid factory" do
+    is_expected.to be_valid
+  end
+
+  it "is a User" do
+    is_expected.to be_a_kind_of User
+  end
+
+  describe "associations" do
```

Comment:
phân biệt describe và it

Reply:
Phương thức `describe` tạo một nhóm example, trong block `describe` chúng ta có thể khai báo các nested group sử dụng describe hoặc context. Con `it` thì dùng để khai báo một example.